RECOMMENDED.md

```diff
@@ -1 +1 @@
-GNOME is the only desktop that secures privileged wayland protocols like screencopy. This means that on non-GNOME systems, applications can access screen content of the entire desktop. This implicitly includes the content of other applications. It's primarily for this reason that GNOME images are recommended.
\ No newline at end of file
+GNOME is currently the only desktop that secures privileged wayland protocols like screencopy. KDE and Sway both have attemps at fixing this, but do not have a working implementation. <sup>[KDE](https://invent.kde.org/plasma/xdg-desktop-portal-kde/-/issues/7)</sup> <sup>[Sway](https://github.com/swaywm/sway/issues/2333)</sup> This means that on non-GNOME systems, applications can access screen content of the entire desktop. This implicitly includes the content of other applications. It's primarily for this reason that GNOME images are recommended.
```

Comment:
This is not really accurate. They have proposals/open issues, but not any fixes yet.